README.org

```diff
@@ -73,9 +73,17 @@ This is the minimal configuration, and will work with any completing-read compli
 
 *** Embark
 
-Citar will automatically integrate with Embark if it is installed, offering contextual access to actions in the minibuffer and at-point.
+The =citar-embark= package adds contextual access actions in the minibuffer and at-point.
 When using Embark, the Citar actions are generic, and work the same across org, markdown, and latex modes.
 
+#+BEGIN_SRC emacs-lisp
+(use-package citar-embark
+  :bind (("M-;" . embark-act)
+         ("M-," . embark-dwim))
+  :custom
+  (citar-embark 1))
```

Comment:
I think we need to decide on the autoload strategy. As I see it, there are two options:

1. Load `citar-embark` after both `citar` and `embark` are loaded. Then `embark-act` will load Embark, but not detect in-buffer citations unless Citar also gets loaded somehow (either through a mode hook, or using a Citar command).
2. Load `citar-embark` as soon as `embark` is loaded, which will also pull in `citar`. This will "just work" because running any Embark command will also load and activate Citar, but may annoy people who just want to use Embark and aren't doing anything with Citar yet.

In either case, I think it's better to have separate `use-package` stanzas for `citar`, `embark`, and `citar-embark`. This would be the `citar-embark` configuration for option 1:
```elisp
(use-package citar-embark
  :after citar embark                   ; drop "citar" for option 2
  :no-require
  :config (citar-embark-mode))
```
The equivalent without use-package is just
```elisp
(with-eval-after-load 'embark
  (with-eval-after-load 'citar
    (citar-embark-mode)))
```

Reply:
> I think we need to decide on the autoload strategy.

Do you mean just WRT to config? Or does this impact the code base?

If the former, and this is just for the README, we could include both options?

Or add an explanation?